introduction.md

```diff
@@ -0,0 +1 @@
+Hello Sensei, I travel many miles to learn the ways of the git.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.